docs/user/alerting/alerting-setup.asciidoc

```diff
@@ -17,17 +17,17 @@ If you are using an *on-premises* Elastic Stack deployment:
 
 If you are using an *on-premises* Elastic Stack deployment with <<using-kibana-with-security, *security*>>:
```

Comment:
The section starts with "The Alerting feature". Should this just be "Alerting"?

Reply:
Updated